tests/acceptance/test_organization_settings.py

```diff
@@ -19,46 +19,3 @@ def load_organization_helper(self, snapshot_name=None):
         if snapshot_name is not None:
             self.browser.snapshot("organization settings -- " + snapshot_name)
         assert self.browser.element_exists(".ref-organization-settings")
-
-    def renders_2fa_setting(self):
-        return self.browser.element_exists("#require2FA")
-
-    def test_renders_2fa_setting_for_owner(self):
-        user_owner = self.create_user("owner@example.com")
-        organization = self.create_organization(name="Example", owner=user_owner)
-        self.login_as(user_owner)
-        path = "/organizations/%s/settings/" % organization.slug
-
-        self.browser.get(path)
-        self.load_organization_helper()
-        assert self.renders_2fa_setting()
-
-    def test_renders_2fa_setting_for_manager(self):
-        user_manager = self.create_user("manager@gexample.com")
-        organization = self.create_organization(
-            name="Example", owner=self.create_user("owner@example.com")
-        )
-        self.create_member(organization=organization, user=user_manager, role="manager")
-        self.login_as(user_manager)
-        path = "/organizations/%s/settings/" % organization.slug
-
-        self.browser.get(path)
-        self.load_organization_helper()
-        assert self.renders_2fa_setting()
-
-    def test_setting_2fa_without_2fa_enabled(self):
-        user_owner = self.create_user("owner@example.com")
-        organization = self.create_organization(name="Example", owner=user_owner)
-        self.login_as(user_owner)
-        path = "/organizations/%s/settings/" % organization.slug
-
-        self.browser.get(path)
-        self.browser.wait_until_not(".loading-indicator")
-        assert not self.browser.element_exists(".ref-organization-settings .error")
-        self.browser.click("#require2FA")
-
-        self.browser.wait_until(".modal")
-        self.browser.click('.modal [data-test-id="confirm-button"]')
-        self.browser.wait_until_not(".modal")
-        self.browser.wait_until_test_id("toast-error")
-        self.load_organization_helper("setting 2fa without 2fa enabled")
```

Comment:
After this removal there are no tests left in the file.

Suggestion: rename this file to `test_organization_security_privacy.py`, since that's all it was testing.

Reply:
oh I see.. I thought the code below was a test 

```
   def load_organization_helper(self, snapshot_name=None):
        self.browser.wait_until_not(".loading-indicator")
        if snapshot_name is not None:
            self.browser.snapshot("organization settings -- " + snapshot_name)
        assert self.browser.element_exists(".ref-organization-settings")
```